spec/rubocop/cop/rails5/no_string_default_for_json_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails5::NoStringDefaultForJson, :config do
+  let(:config) { RuboCop::Config.new }
+
+  it 'registers an offense when using string as default value for json field on create_table' do
+    expect_offense(<<~RUBY)
+      t.json :archive_data, default: '{}'
+      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Do not use string as a default value for JSON or JSONB columns. Column: archive_data
+    RUBY
+
+    expect_offense(<<~RUBY)
+      t.jsonb :archive_data, default: '[]'
+      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Do not use string as a default value for JSON or JSONB columns. Column: archive_data
+    RUBY
+  end
+
+  it 'registers an offense when using string as default value for json field on add_column/change_colum' do
+    expect_offense(<<~RUBY)
+      add_column :appointments, :more_data, :json, default: '[]'
+      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Do not use string as a default value for JSON or JSONB columns. Column: more_data
+    RUBY
+
+    expect_offense(<<~RUBY)
+      change_column :questions, :archive_data, :jsonb, default: '{}'
+      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Do not use string as a default value for JSON or JSONB columns. Column: archive_data
+    RUBY
+  end
```

Comment:
Can it test that the error is not matched, when the default is value is not set, is `{}` or `[]`?

Reply:
@lucasdavila thanks for the review. I added tests for [objects as default](https://github.com/telus-health-ch/rubocop-chr/pull/3/commits/2ce71e86d4a504f7dc842616182b8118081f9905) and for [no default value defined](https://github.com/telus-health-ch/rubocop-chr/pull/3/commits/e4f7d42ebaa842369cfd3f530fb1c17ab7ddf8b0).

Reply:
Thanks 🙌🏻